__tests__/app.test.js

```diff
@@ -246,3 +246,46 @@ describe('POST /api/articles/:article_id/comments', () => {
       });
   });
 });
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("200: Successfully updates the votes of a valid article", () => {
+      return request(app)
+          .patch('/api/articles/1')
+          .send({ inc_votes: 10 })
+          .expect(200)
+          .then(({ body: { article } }) => {
+              expect(article).toHaveProperty('article_id', 1);
+              expect(article).toHaveProperty('votes', expect.any(Number));  
+          });
```

Comment:
This is a new endpoint so we should be testing for the entire article object instead of just two properties